streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -36,21 +37,34 @@ public ClientState() {
     }
 
     ClientState(final int capacity) {
-        this(new HashSet<TaskId>(), new HashSet<TaskId>(), new HashSet<TaskId>(), new HashSet<TaskId>(), new HashSet<TaskId>(), capacity);
+        this(new HashSet<TaskId>(), new HashSet<TaskId>(), new HashSet<TaskId>(), new HashSet<TaskId>(), new HashSet<TaskId>(), new HashSet<TaskId>(), capacity);
     }
 
-    private ClientState(Set<TaskId> activeTasks, Set<TaskId> standbyTasks, Set<TaskId> assignedTasks, Set<TaskId> prevActiveTasks, Set<TaskId> prevAssignedTasks, int capacity) {
+    private ClientState(final Set<TaskId> activeTasks,
```

Comment:
The added `prevStandbyTasks` seems not set anywhere? I.e. it will always be empty hashset?

Reply:
It is set in 
```
public void addPreviousStandbyTasks(final Set<TaskId> standbyTasks) {
    prevStandbyTasks.addAll(standbyTasks);
    prevAssignedTasks.addAll(standbyTasks);
}
```